docs/data/base/components/trap-focus/trap-focus.md

```diff
@@ -84,3 +84,9 @@ When auto focus is disabled—as in the demo below—the component only traps th
 The following demo uses the [`Portal`](/base/react-portal/) component to render a subset of the `TrapFocus` children into a new "subtree" outside of the current DOM hierarchy, so they are no longer part of the focus loop:
 
 {{"demo": "PortalTrapFocus.js"}}
+
+## Using a toggle inside the trap
```

Comment:
Let's make this an h3 so it's nested under the Customization h2.

```suggestion
### Using a toggle inside the trap
```

Reply:
Ah, fantastic catch! Looks like that [changed since I opened this PR](https://github.com/mui/material-ui/commit/d207a3b734c0130248179b5d88a5701b123f68c5#diff-d58236350be1411fc5eaaf3d684738c9d7a5e38f95b0b16c58edc97e9fceb95cR63), and I missed that when I merged 😅